.eslintignore

```diff
@@ -32,8 +32,11 @@ package-lock.json
 !/resources/todomvc/vanilla-examples/javascript-es6-webpack
 !/resources/todomvc/vanilla-examples/javascript-es6-webpack-complex
 !/resources/todomvc/vanilla-examples/javascript-web-components
+!/resources/todomvc/vanilla-examples/javascript-web-components-complex/
 
 !/resources/todomvc/big-dom-generator
+resources/todomvc/big-dom-generator/angular
+resources/todomvc/big-dom-generator/javascript-web-components
```

Comment:
why are we ignoring these?

Reply:
These are the generated files, I'm trying to produce them formatted but I think that if the formatter ends up changing them, it might be confusing, WDYT?

Reply:
That makes sense